pom.xml

```diff
@@ -849,7 +860,7 @@
                     </executions>
                     <configuration>
                         <mainClass>${start-class}</mainClass>
-                        <fork>true</fork>
+                        <fork>false</fork>
```

Comment:
Changed it because I would meet error when building the project, the error is below:
![image](https://user-images.githubusercontent.com/32425638/98025371-333cf200-1dcf-11eb-8424-da3218c58ee3.png)
It's the solution I googled from slackoverflow

Reply:
This is new to me. Maybe related to the dependencies added in this pr.